src/components/tree-detail/tree-age.tsx

```diff
@@ -14,7 +14,7 @@ export const TreeAge: React.FC<TreeAgeProps> = ({ treeAge }) => {
 				<TreeAgeIcon />
 				<div className="">{i18n.treeDetail.ageTitle}</div>
 			</div>
-			{treeAge ? (
+			{treeAge !== undefined && treeAge !== null ? (
```

Comment:
```
{treeAge ? (…) }
```

Reply:
`treeAge ? (...)` is not correct in this case because it evaluates to `false` if `treeAge=0 (integer)`, that's why we now explicit test for undefined and null.